compass_app/app/test/data/repositories/activity/activity_repository_remote_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:compass_app/data/repositories/activity/activity_repository.dart';
```

Comment:
Add copyright header to *all* dart files.

Reply:
Done